### PR TITLE
[AIRFLOW-4260] Fix sphinx deprecation warnings

### PR DESCRIPTION
**Pull request by feluelle**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-4260
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

Since sphinx version 1.8 `autodoc_default_flags` is deprecated and can be replaced by using `autodoc_default_options` instead.

See: http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_default_flags (and right below this option is the new one)

Since sphinx version 1.8 `viewcode_import` is deprecated. It has been renamed to `viewcode_follow_imported_members`

See https://www.sphinx-doc.org/en/stable/usage/extensions/viewcode.html#confval-viewcode_follow_imported_members

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

**Comment by XD-DENG** [comment]
Thanks @feluelle . I noticed that there is another doc error due to deprecation `/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/sphinx/events.py:76: RemovedInSphinx30Warning: viewcode_import was renamed to viewcode_follow_imported_members. Please update your configuration.` (https://travis-ci.org/apache/airflow/jobs/517586747)

Would you like to fix it together here?

File: docs/conf.py [hunk 1]
```diff
@@ -122,7 +122,10 @@
     'docroles'
 ]
 
-autodoc_default_flags = ['show-inheritance', 'members']
+autodoc_default_options = {
+    'show-inheritance': True,
+    'members': True
+}
 
 viewcode_import = True
```
Comment (XD-DENG) [inline docs/conf.py]: This line is what I was talking about.

Comment (feluelle) [reply docs/conf.py]: Thanks @XD-DENG I added this fix as well.

Comment (XD-DENG) [reply docs/conf.py]: Thanks!

Will merge once CI passes.

**Pull request merged**